Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -26,6 +27,7 @@
 handle_proxy()
 # Make sure our python code doesn't go through a proxy when communicating with chrome webdriver
 os.environ['no_proxy'] = 'localhost,127.0.0.1'
+os.environ['HOME'] = tempfile.gettempdir()
```

Comment:
Add a comment above: # Needed for cases that rasterize is running with non-root user (docker hardening)